packages/lazy-images/.phpcs.xml.dist

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0"?>
```

Comment:
Do we need this custom configuration here? Can we rely on the existing configuration instead?

Reply:
Looking more into this, I now understand better the processes on Jetpack :) I have now removed it so the existing configuration will be use instead.